aws/static-website/modules/cloudfront/README.md

```diff
@@ -5,8 +5,7 @@ Terraform sub-module which creates Cloudfront resources on AWS.
 ### Create Cloudfront
 ```hcl
 module "cloudfront" {
-  source  = "c0x12c/static-website/aws//modules/cloudfront"
-  version = "~> 1.0"
+  source  = github.com/spartan-stratos/terraform-modules//aws/static-website/aws//modules/cloudfront"
```

Comment:
this should be
`source = github.com/spartan-stratos/terraform-modules//aws/static-website/modules/cloudfront?ref=v0.1.0"`